docs/es/index.md

```diff
@@ -56,17 +56,18 @@ Todos los participantes que envíen un proyecto recibirán una insignia digital.
 ### Español
 
 [Regístrate para todas las sesiones en español](https://developer.microsoft.com/reactor/series/S-1512/)
-
-| Día/Hora              | Tema                    | Pista                    | Recursos                |
-| --------------------- | ------------------------ | ------------------------ | ------------------------ |
-| 4/8 12:00 PM PT | [Bienvenidos al Hackathon de Agentes de IA](https://developer.microsoft.com/reactor/events/25341) | Todas | - |
-| 4/16 09:00 AM PT    | [Crea tu aplicación de código con Azure AI Agent Service](https://developer.microsoft.com/reactor/events/25360) | Python | - |
-| 4/16 12:00 PM PT | [Agentes de IA + .NET Aspire](https://developer.microsoft.com/reactor/events/25333) | C# | - |
-| 4/17 09:00 AM PT    | [Construyendo agentes utilizando un ejército de modelos con el catálogo de Azure AI Foundry](https://developer.microsoft.com/reactor/events/25316) | Python | - |
-| 4/17 12:00 PM PT | [Crea aplicaciones de agentes de IA con Semantic Kernel](https://developer.microsoft.com/reactor/events/25340/) | Python | - |
-| 4/22 12:00 PM PT | [Prototipando agentes de IA con GitHub Models](https://developer.microsoft.com/reactor/events/25483/) | Python | - |
-| 4/23 12:00 PM PT    | [Comunicación dinámica en agentes grupales](https://developer.microsoft.com/reactor/events/25339) | Python | - |
-| 4/23 03:00 PM PT | [VoiceRAG: habla con tus datos](https://developer.microsoft.com/reactor/events/25485/) | Python | - |
+¡Si estás buscando los recursos mostrados en cualquiera de las sesiones, [puedes encontrarlos aquí!](https://github.com/microsoft/AI_Agents_Hackathon/discussions/59)
+
+| Día/Hora              | Tema                    | Pista                    |
+| --------------------- | ------------------------ | ------------------------ |
+| 4/8 12:00 PM PT | [Bienvenidos al Hackathon de Agentes de IA](https://developer.microsoft.com/reactor/events/25341) | Todas |
+| 4/16 09:00 AM PT    | [Crea tu aplicación de código con Azure AI Agent Service](https://developer.microsoft.com/reactor/events/25360) | Python |

```

Comment:
It caught my attention that this and the other one are scheduled for the same day. Is that okay? Also, in Spanish-speaking countries, we use the day:month format (example, _16 de Abril_), so people might feel a bit confused otherwise.

Reply:
the discord image is not showing
![image](https://github.com/user-attachments/assets/98b68113-a07c-45af-bc15-785a55d9f081)